src/GraphQL/Execution/ExecutionResult.cs

```diff
@@ -96,8 +96,8 @@ public ExecutionResult AddErrors(ExecutionErrors errors)
         {
             if (errors?.Count > 0)
             {
-                if (Errors == null)
-                    Errors = new ExecutionErrors(errors.Count);
+                Errors ??= new ExecutionErrors(errors.Count);
+
                 foreach (var error in errors.List)
```

Comment:
There should be no problem here. The method is called at the very end of `DocumentExecuter.ExecuteAsync`.